lms/static/certificates/sass/_components.scss

```diff
@@ -569,3 +572,89 @@
     }
 }
 
+// ------------------------------
+// #BADGES MODAL
+// ------------------------------
+.badges-overlay {
+
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    z-index: 10;
+    background-color: rgba(0, 0, 0, 0.5); /*dim the background*/
+    vertical-align: middle;
+
+    .badges-modal {
+        color: #9b9b9b;
+        font-size: 14pt;
```

Comment:
I'd recommend that you use a copy-based class in the HTML to get this effect. Please see - https://github.com/edx/edx-platform/blob/mattdrayer/SOL-465/static%2Fpattern-library%2Fsass%2Fcomponents%2F_copy.scss

If you need to customize this past using a copy-based class, please use an existing font-size from our typography scale Sass map - https://github.com/edx/edx-platform/blob/mattdrayer/SOL-465/static%2Fpattern-library%2Fsass%2Futilities%2F_variables.scss#L194. You can see an example of how we call this here - https://github.com/edx/edx-platform/blob/mattdrayer/SOL-465/static%2Fpattern-library%2Fsass%2Fcomponents%2F_headings.scss#L47